configuration/scripts/tests/unittest_suite.ts

```diff
@@ -6,3 +6,5 @@ unittest       gx3     1x1x25x29x16 sumchk
 unittest       tx1     8x1          sumchk
 unittest       gx3     4x1          bcstchk
 unittest       gx3     1x1          bcstchk
+unittest       gx3     8x2          gridavgchk,dwblockall
+unittest       gx3     8x2          gridavgchk
```

Comment:
General comment about this 'unittest' test suite: We should probably run it as part of the CI, if possible. And maybe, instruct developers/contributors to run it as part of the standard testing procedure (i.e. in addition to the base_suite ) ?